.travis.yml

```diff
@@ -351,4 +351,7 @@ jobs:
 before_install:
   - pip install tox-tags
 script:
+  # keep it like this to first create env and later to run again with tests
+  # in order to avoid regressions to run twice on the same environment.
+  - tox --notest
```

Comment:
Unclear on this. I'd rather not lose more time on our CI by running tox twice (even if tests are skipped first).

Reply:
The slowdown is of matter of less than 5 seconds, but the benefits are clear. Is not the first project where I see this done, but only know I fully realised why is needed. Think about it as idempotency test for tox installation ;)

Reply:
https://github.com/ansible/molecule/pull/2382#pullrequestreview-304255010